README.md

```diff
@@ -1,5 +1,109 @@
-# Codess in the classroom attendee material
+# Codess In The Classroom
 
-Repository to store the specs and instructions to use in the classroom.
+Welcome! This repository contains the Codess In The Classroom course
+materials. From here you will be able to find (and contribute to!) the
+materials we use in our hands-on workshops.
 
-1. [Week 1](./week1-webclient.md)
+## Introduction
+
+The course is going to be structured as a series of workshops, during which
+you will learn about various technologies and practices used in modern
+software development.
+
+In each workshop, you and your peers will be the development team working on
+part of a web application for a pet adoption center. You will learn about
+front-end web development, APIs, testing and continuous integration, as well
+as how to deploy web applications to the cloud, and how to collaborate in a
+modern software team.
+
+During the workshop, you will have access to a group of mentors, who will be
+there to help supplement the course materials and answer any questions you
+may have.
+
+## Learning goals
+
+Our goal is to give you a chance to learn about modern software development
+practices, and especially to learn about how web applications are designed,
+built, tested, and deployed.
+
+Over the duration of the course, you will build all the important parts of a
+web application that allows users to find and adopt adorable furry
+companions. Each workshop will focus on a different part of the application,
+and will draw on a different set of skills.
+
+Below, you'll find out what to expect in each workshop.
+
+## Course outline
+
+### Workshop 1: modern front-end web development
+
+In this workshop, we're going to build a frontend of a website for the pet
+adoption center. In doing this, you'll learn:
+
+- What is a single-page application (SPA)?
+- How to create a SPA using `create-react-app`.
+- How to scaffold a user interface using `react-bootstrap`.
+- What is an API (application programming interface)?
+- How to talk to an API from our React SPA.
+
+We'll also cover:
+
+- How to use the Git version control system to submit a pull request (PR).
+- How to use pull requests as part of a code review process.
```

Comment:
FYI, The process agenda items are possibly more important than the spa bits, because technology and libraries will vary by team and job, but these processes are common in all large scale companies.

Reply:
That's a good point. I'll shuffle these around a little and change the emphasis.